File: csharp/raindrops/Raindrops.cs
using System;
using System.Text;

public static class Raindrops
{
    public static string Convert(int number)
    {
        var sb = new StringBuilder();
        
		if(number % 3 == 0) sb.Append("Pling");		
		if(number % 5 == 0) sb.Append("Plang");
		if(number % 7 == 0) sb.Append("Plong");
		
        return sb.Length == 0? number.ToString() : sb.ToString();
    }
}

File: javascript/raindrops/raindrops.js
//
// This is only a SKELETON file for the 'Raindrops' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const convert = (input) => {

  let output = '';

  if(input % 3 == 0) output += 'Pling';		
  if(input % 5 == 0) output += 'Plang';
  if(input % 7 == 0) output += 'Plong';

  return output === '' ? input.toString() : output;
};
